docs/getting-started/integrate/browser.mdx

```diff
@@ -77,6 +77,13 @@ import App from './App'
 if (process.env.NODE_ENV === 'development') {
   const { worker } = require('./mocks/browser')
   worker.start()
+  /*
+  // or you can do this instead
```

Comment:
The problem is that dynamic import works in a different way than `require` does, so they are not interchangeable in any regard. 

- `require` is not statically analyzable so bundlers cannot tree-shake its calls /  `import` can be tree-shaken.
- `require` is sync; `import` is async and must be awaited.

Those are very distinct differences and they directly affect how you should treat setups that utilize either method. 

On a related note, we're discussing an issue in Next where the usage of dynamic import results in a race condition between the worker activation and requests on the initial load (see https://github.com/vercel/next.js/issues/43284?notification_referrer_id=NT_kwDOAOSmz7M0OTA1NjI4NjY0OjE0OTg0OTEx&notifications_query=repo%3Avercel%2Fnext.js#issuecomment-1339929638).